src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs

```diff
@@ -105,11 +99,7 @@ public override ICommandLineParserResult ParseArgs(string[] args)
                 .WithDescription("to use with --cert. Either the password, or a file that contains the password for the pfx file")
                 .Callback(p => CertPassword = p);
 
-            Parser
-                .Setup<DebuggerType>("debug")
-                .WithDescription("Default is None. Options are VSCode and VS")
-                .SetDefault(DebuggerType.None)
-                .Callback(d => Debugger = d);
+            IConfigurationArguments = args.Where(arg => arg.StartsWith('/')).ToArray();
```

Comment:
@mamaso was there a merge issue with your change maybe? I think --debug is needed and it seems that @jeffreyCline had updated it in his commit, but it's not here for some reason.

Reply:
ignore. I'm not too smar